srml/balances/src/tests.rs

```diff
@@ -31,6 +31,37 @@ const ID_1: LockIdentifier = *b"1       ";
 const ID_2: LockIdentifier = *b"2       ";
 const ID_3: LockIdentifier = *b"3       ";
 
+#[test]
+fn weight_fee_range() {
+	// rhs values checked in python model
+	// lhs values call mock/DummyFeeHandler
+	// empty 
+	assert_eq!(DummyFeeHandler::convert(0), 0);
+	// (1) lowest fee correct (one transfer transaction = 28)
+	assert_eq!(DummyFeeHandler::convert(28), 27);
+	// (2) half of the ideal
+	assert_eq!(DummyFeeHandler::convert(512 * 1024), 524285);
+	// (3) 5 below the ideal, between the half and the ideal
+	assert_eq!(DummyFeeHandler::convert((512 * 1024) + 87_381), 611666);
```

Comment:
Useful test but should not --preferably-- fail once we use any of the constants. Should use max and Ideal `const`.